### ISSUE TITLE
remove or tune down the hover effect in the chatlist

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-desktop#1600]
it happens to me several times that i _thought_ some chat is selected, but in reality, the "selection" was just a hovered item - eg. select a chat and scroll down in the list while the mouse is still in the chatlist. when you are then distracted by sth. else and come back to Delta Chat, there is a chatlist with a hilited item (the really visible item is somewhere else) - but that hilited item does not match the chat.

maybe this is only me, however, with the contact-request-gray this can get really confusing. the pinned chat add something more to the confusion. when hovering the _selected item_ it appears as if the hover is gone  (when done correctly, the hover has to be toned also for contact-requests, selection, pinned)

finally, i do not really like this flickering.

so, i am wondering why exactly the hover effect is needed here _at all?_ maybe removing it will solve lots of these issues.

**Comment by Jikstra** [comment]
Yup I also don't like the flickering, especially with the contact requests. And I think it should be a lot more subtle.

**Comment by r10s** [comment]
cool - i think, the easiest way would be to just remove it.

alternatively, if we _really_ want some hover-effect, we could eg. _slightly_ dim the chat-title. but, as this is effort, i would go for just removing it, for now :)

**Comment by Simon-Laux** [comment]
> cool - i think, the easiest way would be to just remove it.

It acts as some kind of feedback that this is clickable.
make it really subtle yes, I agree, but I disagree with removing it. (I already try to fix some of it in #1575)
Also I don't like the contact request (background color).

About the hover effect on general, everyone does it including telegram.
If you personally dislike it really hard you can create a custom theme with #1575 that removes the hover effect altogether.

> when hovering the selected item it appears as if the hover is gone

the color is changing too, but that might be too subtle to notice, also it gets brighter instead of darker in this case.

TL;DR;
I don't see a reason for "killing the hover effect", though I agree that it needs to be adjusted. @r10s Can you try the light theme in #1575 and tell me if the issue appears still there?

**Comment by r10s** [comment]
> About the hover effect on general, everyone does it including telegram.

telegram on mac does not have hover effects. not sure about "everyone" :)

**Comment by Simon-Laux** [comment]
mac os is special

**Comment by Simon-Laux** [comment]
I think there is a reason why so many big apps do this.
I asked a friend what he thinks about those hover colors and he said that he expects it from modern applications and that it would increase the perceived quality of the app.
Another friend believes its helpful, informational.
And the third friend I asked told me that he finds it useful for chat apps such as skype.
(context: all of them are more or less gamers)

**Comment by r10s** [comment]
thanks for your thoughts.

a very slight toning might be an option, for sure, i mentioned that initially. but that requires different toning of all different items backgrounds in the list - multiplied with the themes, this is quite some effort.

keeping that in mind, for a good effort:effect ratio, just removing it also already improve things and might be reasonable therefore.

**Comment by Simon-Laux** [comment]
> multiplied with the themes, this is quite some effort

I think its manageable. We will use scss for theming and there we can "derive" colors. like say we use a 2% darker version for the hover colors. (#1575)
